webextensions/api/declarativeNetRequest.json

```diff
@@ -0,0 +1,1068 @@
+                "chrome": {
+                  "version_added": "88"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            },
+            "tabUpdate": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "89"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15"
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            }
+          }
+        },
+        "testMatchOutcome": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/testMatchOutcome",
+            "support": {
+              "chrome": {
+                "version_added": "103"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "testMatchRequestDetails": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "103"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          }
+        },
+        "updateDynamicRules": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/updateDynamicRules",
+            "support": {
+              "chrome": {
+                "version_added": "84"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "options": {
```

Comment:
@Rob--W because the way the Chrome documentation appears to work is that everything that doesn't have a specific release identified was released in 84 and [the documentation for `updateDynamicRules`](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/#method-updateDynamicRules) shows that `options` with added in 87.

Reply:
@Rob--W given your comments, I have assumed the information about adding `options` in 87 is incorrect and have removed it from the BCD, retaining 84 as the release that introduced `updateDynamicRules`.

Reply:
The API was introduced in 84, but a breaking change to the syntax was introduced in 87.
https://bugs.chromium.org/p/chromium/issues/detail?id=1131746 has details. If you'd like to be complete, you could add that it was supported from 84 - 86 with a different syntax, and 87+ with the current format.

Since it has been over 2 years now, I think that it's fine to mention 87+ as supported version.